app/src/main/java/org/oppia/android/app/player/audio/AudioFragmentPresenter.kt

```diff
@@ -123,7 +123,6 @@ class AudioFragmentPresenter @Inject constructor(
       activity,
       Observer<String> { result ->
         viewModel.selectedLanguageCode = result
-        viewModel.loadMainContentAudio(false)
```

Comment:
@rt4914 
I've tried recreating AudioFragment but it was a little complicated 
later I just removed this line and everything works without any crash

Please check if it works.

Reply:
I guess if it works tihs pr will be 'ready for review'

Reply:
This does work but at the cost of removing the functionality. 
Maybe this link can help. https://stackoverflow.com/questions/6599376/illegalstateexception-calling-mediaplayer-reset

Otherwise once @BenHenning is available we will need to discuss this with him in more detail.